infrastructure/oss/issue_html.go

```diff
@@ -95,6 +95,11 @@ func getIntroducedBy(issue snyk.OssIssueData) string {
 	}
 }
 
+func getVulnHtmlAnchor(packageManager string, module string) string {
+	snykUi := config.CurrentConfig().SnykUi()
+	return fmt.Sprintf("<a href='%s/test/%s'>%s</a>", snykUi, packageManager, module)
```

Comment:
This is a completely unnecessary comment. Just wanted to add that I recently read, that `fmt.Sprintf` uses more heap allocations than `strings.Builder`. So it may be more efficient to use the latter - but in this case performance should not be an issue.

Reply:
Interesting read regarding this: https://stackoverflow.com/questions/1760757/how-to-efficiently-concatenate-strings-in-go/47798475#47798475